ccl/multiregionccl: TestMultiRegionTenantRegions failed

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/fd5d11fc-b539-4263-8ad9-21eaceb3b253?testReportRun=2&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on release-24.3.15-rc @ [b33c70a891bb565033304c0411af9875506051f3](https://github.com/cockroachdb/cockroach/commits/b33c70a891bb565033304c0411af9875506051f3):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMultiRegionTenantRegions837328290
    test_log_scope.go:76: use -show-logs to present logs inline
    multiregion_system_table_test.go:448: error executing query="ALTER DATABASE system SET PRIMARY REGION \"us-east1\"" args=[]: ERROR: dial tcp 127.0.0.1:42727: connect: connection refused
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionTenantRegions837328290
--- FAIL: TestMultiRegionTenantRegions (101.37s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #148416 ccl/multiregionccl: TestMultiRegionTenantRegions failed [failed to count leases] [C-test-failure O-robot P-2 T-sql-foundations branch-master]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-51934

--- a/pkg/ccl/multiregionccl/multiregion_system_table_test.go
+++ b/pkg/ccl/multiregionccl/multiregion_system_table_test.go
@@ -330,6 +330,8 @@ func TestMultiRegionTenantRegions(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDuress(t, "slow test")
+
 	tc, _, cleanup := multiregionccltestutils.TestingCreateMultiRegionCluster(
 		t, 3 /*numServers*/, base.TestingKnobs{},
 	)

